VideoWeb/VideoWeb/Extensions/ConfigureServicesExtensions.cs

```diff
@@ -255,7 +255,10 @@ private static IAsyncPolicy<HttpResponseMessage> GetCircuitBreakerPolicy()
         private static IVideoApiClient BuildVideoApiClient(HttpClient httpClient,
             HearingServicesConfiguration serviceSettings)
         {
-            return VideoApiClient.GetClient(serviceSettings.VideoApiUrl, httpClient);
+            return new VideoApiClient(httpClient)
+            {
```

Comment:
is this correct?

Reply:
Yup, I tested this locally and the correct URL was assigned

Reply:
reverting to the factory method to maintain consistency